src/test/kotlin/org/rust/ide/inspections/RsAssertEqualInspectionTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+class RsAssertEqualInspectionTest : RsInspectionsTestBase(RsAssertEqualInspection()) {
+    fun testSimpleAssertEqFix() = checkFixByText("Convert to assert_eq!", """
```

Comment:
For new code, we use space separated test names, like `test simple assert eq fix`.